content/english/Kusto-KC7/activity-2.md

```diff
@@ -0,0 +1,215 @@
+    {
+      s = quizForm.elements[i].value;
+    }
+  }
+
+  // no choice was selected
+  //
+  if("?" == s)
+  {
+    alert("Please make a selection.");
+    return false;
+  }
+
+  // check if we have the correct
+  // choice selected
+  //
+  if(s == theAnswer)
+  {
+    alert("'"+s+"' is correct!");
+    if(urlRight)
+    {
+    document.location.href = urlRight;
+    }
+  }
+  else
+  {
+    alert("'"+s+"' is incorrect.");
+    if( urlWrong )
+    {
+    document.location.href = urlWrong;
+    }
+  }
+
+  // return "false" to indicate not to
+  // submit the form.
+  // change this to "true" if the form
+  // "action" is valid,
+  // i.e. points to a valid CGI script
+  //
+  return false;
+}
+//-->
+</SCRIPT>
+
+{{< /rawhtml >}}
+
+
+
+{{< alert theme="success" >}}**Best way to complete this section is to play the video while following along with the page below**{{< /alert >}}
+
+### What is Attribution?
+
+One key topic in the world of cybersecurity you'll come across is called **attribution**. Attribution in cybersecurity refers to the process of tracking, identifying, and generally attributing blame on the perpatrator of a cyber attack. Cyber attacks, also known commonly as *hacking*, are an attempt by an adversary to gain access to systems for the purpose of altering, stealing, destroying, or exposing information. Examples of methods to carry out a cyber attack include:
+- **Malware** - Short for malicious software, is any code created with intent to do harm like a virus or a worm
+- **Phishing** - An attack that uses email, text messaging (SMS), or social media to entice a victim to share sensitive information or to download a malicious file, this is much like catfishing
+- **Ransomware** - Malicious code intended to lock victims out of their own systems with in order to exact a ransom, usually money, from the victim in exchange for regaining access to their locked systems or data
+- **Password Spray#** - An attacker aquires a list of usernames, and then tries logging in across all usernames using the same password, and then repeats the process with new passwords until they gain system access.
+
+### Challenges to Attribution
+
+Although it is a key part of cybersecurity, attribution can be challenging to do, even for cybersecurity experts. Experts often have to conduct extensive forensic investigations and analyse lots of data looking for ways to prove who could be responsible for attacks. Some things experts look at to help with this are:
+- Historical Data - Are there repeated patterns used across multiple cyber attacks over time that could indicate which malicious actor(s) could be behind this?
+- Intent or Motives - Are there attacks specifically targetting educational institutions for example? Could a cyber attack coincide with a publicly observable political conflict between countries that cannot afford to be caught shooting missiles, sending military troops, or otherwise engaging in **kinetic warfare** with another country?
```

Comment:
```suggestion
- Intent or Motives - Are there attacks specifically targeting educational institutions, for example? Could a cyber attack coincide with a publicly observable political conflict between countries that cannot afford to be caught shooting missiles, sending military troops, or otherwise engaging in **kinetic warfare** with another country?
```